extensions/amp-fit-text/0.1/test-e2e/test-amp-fit-text.js

```diff
@@ -23,15 +23,18 @@ describes.endtoend(
   (env) => {
     let controller;
 
+    before(() => {
```

Comment:
```suggestion
  before(function () {
```

Looks like the test failures in the most recent run are having issues with this anonymous fn.

Reply:
Aha, we can't use `this` in an anonymous function. Gotcha.

Reply:
That didn't work either, because E2E tests don't use the same stack as integration tests, so they don't load `_init_tests.js`, and don't have access to the `this.skipTest()` helper. I've rewritten this and made sure it works.

![image](https://user-images.githubusercontent.com/26553114/107587916-aa550400-6bd0-11eb-8328-7366e5680ca2.png)